ui/app/models/node-pool.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Model from '@ember-data/model';
+import { attr } from '@ember-data/model';
+import { computed } from '@ember/object';
+import classic from 'ember-classic-decorator';
+
+@classic
```

Comment:
Does it need to be a classic model or can we start to leverage Octane for new models?

Reply:
We're stuck on `classic` because we rely on base models to extend that are `classic`. We will likely stay on `classic` for the foreseeable future.

Reply:
Ah got it, thanks!